pom.xml

```diff
@@ -87,6 +91,11 @@ SOFTWARE.
       <artifactId>snakeyaml</artifactId>
       <version>1.33</version>
     </dependency>
+    <dependency>
+      <groupId>org.cactoos</groupId>
+      <artifactId>cactoos</artifactId>
+      <version>0.55.0</version>
```

Comment:
@volodya-lombrozo `scope` should be `test`?

Reply:
@yegor256 sure, done